packages/subapp-web/lib/init.js

```diff
@@ -34,14 +29,28 @@ module.exports = function setup(setupContext) {
     basePath: ""
   };
 
+  let inlineRuntimeJS = "";
+  let runtimeEntryPoints = [];
+  if (process.env.NODE_ENV === "production") {
+    runtimeEntryPoints = Object.keys(cdnJsBundles).filter(ep =>
+      cdnJsBundles[ep].includes("runtime.bundle")
+    );
+    inlineRuntimeJS = runtimeEntryPoints
+      .map(ep => Path.resolve("dist", "js", Path.basename(cdnJsBundles[ep])))
+      .filter(fullPath => Fs.existsSync(fullPath))
+      .map(fullPath => Fs.readFileSync(fullPath))
+      .join(" ");
+  }
+
   const webSubAppJs = `<script id="bundleAssets" type="application/json">
 ${JSON.stringify(bundleAssets)}
 </script>
 <script>/*LJ*/${loadJs}/*LJ*/
+/*rt*/${inlineRuntimeJS} /*rt*/
 ${clientJs}
 ${cdnJs}
-</script>
-`;
+window.xarcV1.markBundlesLoaded([${runtimeEntryPoints.join(",")}]);
```

Comment:
what if `runtimeEntryPoints` has nothing?

Reply:
it would call ```window.xarcV1.markBundlesLoaded([]);``` which doesn't seem to throw an exception.

Reply:
I changed to using ```window.xarcV1.markBundlesLoaded(${JSON.stringify(runtimeEntryPoints)});
```